tutorials/2d/custom_drawing_in_2d.rst

```diff
@@ -211,7 +211,7 @@ We first determine the angle of each point, between the starting and ending angl
 The reason why each angle is reduced by 90° is that we will compute 2D positions
 out of each angle using trigonometry (you know, cosine and sine stuff...). However,
 to be simple, cos() and sin() use radians, not degrees. The angle of 0° (0 radian)
-starts at 3 o'clock although we want to start counting at 12 o'clock. So we reduce
+starts at 3 o'clock, although we want to start counting at 12 o'clock. So we reduce
```

Comment:
Suggestion (here and above): "reduce" → "decrease", as reduction is something else in mathematics.
[Wiktionary: reduce](https://en.wiktionary.org/wiki/reduce#Verb): (mathematics) To simplify an equation or formula without changing its value.